tools/SetupFlow/DevHome.SetupFlow/TaskGroups/AppManagementTaskGroup.cs

```diff
@@ -39,11 +39,9 @@ public class AppManagementTaskGroup : ISetupTaskGroup
 
     public void HandleSearchQuery(string query)
     {
-        var searchParameter = HttpUtility.ParseQueryString(query)["search"];
-        if (!string.IsNullOrEmpty(searchParameter))
+        if (!string.IsNullOrEmpty(query))
         {
-            var trimmedSearchParameter = searchParameter.Trim('\"');
-            _appManagementViewModel.PerformSearch(trimmedSearchParameter);
+            _appManagementViewModel.PerformSearch(query);

```

Comment:
The only action `PerformSearch` is doing in setting a public property.  No need for a set method unless the code needs to perform additional actions.

Reply:
I think this triggers ` SearchTextChangedAsync` which handles displaying the shimmer and has some additional logic beyond just calling SearchAsync. The _searchText I'm setting with this is the private prop in AppManagementViewModel.